bigquery/samples/tests/test_get_table.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import create_table
+from .. import get_table
+
+
+def test_get_table(capsys, client, table_id):
+
+    get_table.get_table(client, table_id)
+    out, err = capsys.readouterr()
+    assert "Got table '{}'.".format(table_id) in out
+    assert "full_name" in out  # test that schema is printed
+    assert "Table description: Sample Table" in out
```

Comment:
A best practice for testing is that any needed test data should appear in the test itself. From [What makes a good test?](https://testing.googleblog.com/2014/03/testing-on-toilet-what-makes-good-test.html):

> A test is complete when its body contains all of the information you need to understand it, and concise when it doesn't contain any other distracting information.

In this case, we should set up the table description in the test itself. The most efficient way to do that here is to use the `random_table_id` fixture and call `client.create_table(...)` with the desired `Table` object before running the sample code.

Reply:
Removed details from fixture and added them to the specific test as directed.